sdk/keyvault/azure-security-keyvault-certificates/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Release History
 
+## 4.1.0-beta.2 (Unreleased)
```

Comment:
@JimSuplizio @praveenkuttappan something in this automation looks wrong can you guys take a look?

Reply:
@weshaggard and @praveenkuttappan, the automation isn't wrong, the CHANGELOG is. We saw some similar issues with badly formatted CHANGELOGs this release. The reason why you're seeing 3 entries is because of "4.2.0-beta.2 (2020-04-09)" is actually incorrect, it should have been 4.1.0-beta.2 (2020-04-09). The reason you're seeing 3 entries is because each commit added a 'missing' entry based upon what was in their respective version files at the time. 
@g2vinay manually updated this version as part of https://github.com/Azure/azure-sdk-for-java/pull/9758 which, unfortunately, wasn't the correct version.